sentinel.go

```diff
@@ -576,8 +614,10 @@ func parseSlaveAddrs(addrs []interface{}) []string {
 		}
 
 		for _, flag := range flags {
-			switch flag {
-			case "s_down", "o_down", "disconnected":
```

Comment:
Please keep this switch but add a separate clause like:

```
case "disconnected":
    if !keepDisconnected {
        isDown = true
    }
```

Reply:
I fix it.